examples/tts/conf/tacotron2_44100.yaml

```diff
@@ -1,116 +1,137 @@
-# TODO(Oktai15): update this config in 1.8.0 version
+# This config contains the default values for training Tacotron2 model on LJSpeech dataset.
```

Comment:
Why do we still say LJSpeech here?

Reply:
Specifically because of the pitch min/max statistics that are in there as a default. The other configs also have the same default statistics; do you think we should update them across the board to use the function default or have the user input something?

Reply:
The pitch fmin and fmax no longer change per model. Tacotron2 isn't pitch conditioned anyways so why do we have this?

Reply:
Ah, good point--I'll remove it from both configs.

What do you mean by it no longer changes per model? The TTSDataset takes them in for `add_pitch()`, is this deprecated? https://github.com/NVIDIA/NeMo/blob/71c51ec24d0b2774dcdd7dcbbf91c6875ce5383b/nemo/collections/tts/torch/data.py#L358-L359

Reply:
Ahh, gotcha. Pushing updates.

At some point we should update the other configs to remove these values as well, I'll make a note of it.